### PR TITLE
added theme option to pageConfig

### LINKED ISSUE #2792: [Bug]: meta theme-color is hard coded in _template
### What happened?

```html
<meta name="theme-color" content="#206095"/>
``` 
this is causing this behaviour in safari for IOS devices where the status bar and the overflow scroll is set to blue. 

image-2023-08-10-21-27-01-879

### What was the expected outcome?

We need to update the _template to accept the theme color from the page.congif object. 

### How is this problem impacting your service?

Undesired blue bar displaying on IOS in safari. 

### What list of steps can be followed to reproduce the bug or issue?

open the IDS website on iphone in safari .

### Example

_No response_

### What version of the ONS Design System are you using?

62.0.0

### What devices are you seeing the problem on?

- [ ] Desktop
- [X] Mobile
- [ ] Tablet

### Device details

Iphones

### What operating systems are you seeing the problem on?

- [ ] Windows
- [ ] macOS
- [ ] Android
- [X] iOS
- [ ] Other

### Operating system details

_No response_

### What browsers are you seeing the problem on?

- [ ] Firefox
- [ ] Chrome
- [X] Safari
- [ ] Microsoft Edge
- [ ] Microsoft Internet Explorer
- [ ] Other

### Browser details

_No response_

### PR DESCRIPTION
**Pull request by alessioventuriniAND**
### What is the context of this PR?
This fixes #2792 

### How to review
run the design system and open in on your Iphone in safari. Set the theme option in pageConfig in our page template examples to confirm that the status bar color is changing. 

**We will also need to update the documentation in the service manual to add the them option in the pageConfig object.**

### CODE REVIEW

**Review by precious-onyenaucheya-ons — APPROVED** [review]

File: src/foundations/page-template/example-basic.njk [hunk 1]
```diff
@@ -21,4 +21,4 @@ layout: ~
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
     <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
     <p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-{% endblock %}
+{% endblock %}
```
Comment (rmccar) [inline src/foundations/page-template/example-basic.njk:24]: The line on the end of this file shouldnt really have been removed

File: src/layout/_template.njk [hunk 2]
```diff
@@ -55,7 +59,7 @@
         <title>{{ page_title }}</title>
         <link rel="stylesheet" href="{{ assetsURL }}/css/main.css">
         <link rel="stylesheet" media="print" href="{{ assetsURL }}/css/print.css">
-        <meta name="theme-color" content="#206095"/>
+        <meta name="theme-color" content="{{theme}}"/>
```
Comment (rmccar) [inline src/layout/_template.njk:62]:
```suggestion
        <meta name="theme-color" content="{{ theme }}"/>
```

File: src/layout/_template.njk [hunk 3]
```diff
@@ -128,18 +133,18 @@
                             method="{{ form.method | default('POST') }}"
                             {% if form.attributes %}{% for attribute, value in (form.attributes.items() if form.attributes is mapping and form.attributes.items else form.attributes) %}{{ attribute }}{% if value %}="{{value}}" {% endif %}{% endfor %}{% endif %}
                         >
-                    {% endif %}
-                    {% block preHeader %}{% endblock %}
-                    {% block skipLink %}
-                        {{
+                        {% endif %}
```
Comment (rmccar) [inline src/layout/_template.njk:136]: These code formatting changes should be reverted

File: src/layout/_template.njk [hunk 4]
```diff
@@ -243,4 +248,4 @@
 
         {% block scripts %}{% endblock %}
     </body>
-</html>
+</html>
```
Comment (rmccar) [inline src/layout/_template.njk:251]: This line at the end shouldnt have been removed

File: src/layout/_template.njk [hunk 5]
```diff
@@ -43,9 +44,12 @@
...
 {# Page columns #}
 {% set pageColNumber = pageConfig.pageColNumber | default("12") %}
-{% set pageColClasses = pageConfig.pageColClasses if pageConfig.pageColClasses else '' %}
-{% set mainClasses = pageConfig.mainColClasses if pageConfig.mainColClasses else '' %}
-
+{% set pageColClasses = pageConfig.pageColClasses if pageConfig.pageColClasses else 
+    '' %}
+{% set mainClasses = pageConfig.mainColClasses if pageConfig.mainColClasses else 
+    '' %}
+{% set theme = pageConfig.theme if pageConfig.theme else
```
Comment (rmccar) [inline src/layout/_template.njk:51]: I think the theme param name should be more specific. Maybe `themeColour`? Also this param will need to be documented in the service manual

**Review by rmccar — COMMENTED** [review]
Few minor comments

File: src/foundations/page-template/example-basic.njk (hunk 1 above)
Comment (alessioventuriniAND) [reply src/foundations/page-template/example-basic.njk:24]: For some reason git is showing the line being removed and then added back in the same place. So looks like the line is still there

File: src/layout/_template.njk (hunk 4 above)
Comment (alessioventuriniAND) [reply src/layout/_template.njk:251]: For some reason git is showing the line being removed and then added back in the same place. So looks like the line is still there

File: src/layout/_template.njk (hunk 5 above)
Comment (alessioventuriniAND) [reply src/layout/_template.njk:51]: I have changed this to themeColour :) and yes the documentation on the service manual will need to be updated :)

File: src/foundations/page-template/example-basic.njk (hunk 1 above)
Comment (rmccar) [reply src/foundations/page-template/example-basic.njk:24]: Ah weird

**Review by rmccar — APPROVED** [review]

**Pull request merged**